libr/bin/format/objc/mach0_classes.h

```diff
@@ -0,0 +1,25 @@
+#include <r_bin.h>
+
+#undef Mach0_class_
+#undef Mach0_struct_
+#undef mach0_ut
+#undef r_bin_plugin_mach
+
+#ifdef R_MACH0_CLASS64
+#define Mach0_class_(name) Mach0_class64_##name
```

Comment:
also maybe move mach0_ut and r_bin_plugin_mach to mach0_classes? and than use just R_BIN_MACH064 define and remome my define?